matminer/featurizers/site.py

```diff
@@ -172,38 +186,28 @@ class OPSiteFingerprint(BaseFeaturizer):
                             default: True).
     """
 
-    def __init__(self, optypes=None, dr=0.1, ddr=0.01, ndr=1, dop=0.001,
+    def __init__(self, targets=None, dr=0.1, ddr=0.01, ndr=1, dop=0.001,
```

Comment:
We often use targets to mean target elements to bond to, e.g. the some of the NN codes do this. Maybe ``target_motifs``?